<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Administrators-MacBook-Pro-2.local" name="com.amazon.DriverScripts.TestCaseExecutionScript" tests="5" failures="0" timestamp="18 Sep 2020 10:05:31 GMT" time="122.939" errors="1">
  <testcase name="verifyProductPriceAtCheckoutFunctionality" time="0.732" classname="com.amazon.DriverScripts.TestCaseExecutionScript">
    <error type="org.junit.ComparisonFailure" message="expected:&lt;[]82,990&gt; but was:&lt;[   ]82,990&gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[]82,990> but was:<[   ]82,990>
	at org.junit.Assert.assertEquals(Assert.java:125)
	at org.junit.Assert.assertEquals(Assert.java:147)
	at com.amazon.PagesClass.VerifypriceofproductatcheckoutPage.verifyProductPriceAtCheckoutFunctionality(VerifypriceofproductatcheckoutPage.java:45)
	at com.amazon.DriverScripts.TestCaseExecutionScript.verifyProductPriceAtCheckoutFunctionality(TestCaseExecutionScript.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- verifyProductPriceAtCheckoutFunctionality -->
  <testcase name="loginIntoAmazonApp" time="11.338" classname="com.amazon.DriverScripts.TestCaseExecutionScript"/>
  <testcase name="selectRandomProductFromLListFunctionality" time="86.474" classname="com.amazon.DriverScripts.TestCaseExecutionScript"/>
  <testcase name="verifylogiFunctionality" time="12.250" classname="com.amazon.DriverScripts.TestCaseExecutionScript"/>
  <testcase name="searchProductFromAppFunctionality" time="12.145" classname="com.amazon.DriverScripts.TestCaseExecutionScript"/>
</testsuite> <!-- com.amazon.DriverScripts.TestCaseExecutionScript -->
